engine/api/platform/dao_model.go

```diff
@@ -0,0 +1,131 @@
+package platform
+
+import (
+	"database/sql"
+	"encoding/json"
+
+	"github.com/go-gorp/gorp"
+
+	"github.com/ovh/cds/sdk"
+	"github.com/ovh/cds/sdk/log"
+)
+
+var (
+	Models = []sdk.PlatformModel{
+		sdk.KafkaPlatform,
+	}
+)
+
+// LoadModel Load a platform model by its ID
+func LoadModel(db gorp.SqlExecutor, modelID int64) (sdk.PlatformModel, error) {
+	var pm PlatformModel
+	if err := db.SelectOne(&pm, "SELECT * from platform_model where id = $1", modelID); err != nil {
+		return sdk.PlatformModel{}, sdk.WrapError(err, "LoadModel> Cannot select platform model %d", modelID)
+	}
+	return sdk.PlatformModel(pm), nil
+}
+
+// CreatePlatformModels create platforms models
+func CreatePlatformModels(db *gorp.DbMap) error {
```

Comment:
what about using a repository pattern like in the services package (just asking), dunno it's this king of pattern is really a good approach